src/modules/dashboard/components/ColonyHome/ColonyHome.tsx

```diff
@@ -298,10 +316,13 @@ const ColonyHome = ({
           <BreadCrumb elements={crumbs} />
         </div>
         <Tabs>
-          <TabList>
-            <Tab>
+          <TabList extra={activeTab === 'tasks' ? filterSelect : noFilter}>
```

Comment:
I don't think there's supposed to be a `filterSelect` here when `'tasks'` is the `activeTab`... according to the figma spec it should just be `null`.

Notice the duplicate one in the `extra` position in this screenshot:

<img width="1500" alt="Screen Shot 2019-09-20 at 4 11 01 PM" src="https://user-images.githubusercontent.com/3052635/65359185-65869280-dbc1-11e9-80c7-615e041c431b.png">

Reply:
Of couurse 🥇